packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -64,7 +63,11 @@ const Map<String, String> _kManuallyPinnedDependencies = <String, String>{
   // https://github.com/dart-lang/build/issues/2772
   'build_runner_core': '5.2.0',
   'build_modules': '2.10.1',
-  'path_provider': '1.6.14'
+  'path_provider': '1.6.14',
```

Comment:
why are we pinning `path_provider`?
(just asking because i'm adding comments to `update_packages.dart`)

Reply:
I'm not sure why `path_provider` was pinned. maybe @jonahwilliams ?